data_model/bin/mock-server.js

```diff
@@ -0,0 +1,24 @@
+import fake from '../utils/mapping-faker';
+import createVariants from '../utils/createVariants';
+import { range } from 'lodash';
+import {argv} from 'yargs';
+let model_properties = require(`../model/properties.mapping`).default();
+
+let main = async => {
```

Comment:
same, if we can use const use const, its more like final than an actual constant in JS